src/wrappers.rs

```diff
@@ -364,14 +364,14 @@ macro_rules! py_wrap_simple_enum {
 ///         // Used to implement `TryFrom<P> for PyFoo`. Any errors returned must be `PyErr`.
 ///         py -> rs {
 ///             py_dict: Py<PyDict> => Foo {
-///                 let bar = py_dict.as_ref(py).get_item("bar").unwrap().extract().unwrap();
-///                 let baz = py_dict.as_ref(py).get_item("baz").unwrap().extract().unwrap();
+///                 let bar = py_dict.as_ref(py).get_item("bar")?.unwrap().extract().unwrap();
```

Comment:
```suggestion
///                 let bar = py_dict.as_ref(py).get_item("bar")?.extract().unwrap();
```

Reply:
In 0.20.0, [`get_item`](https://docs.rs/pyo3/latest/pyo3/types/struct.PyDict.html#method.get_item) returns a `PyResult<Option<&PyAny>>` instead of just an `Option<&PyAny>`, so we need do actually need both the `?` and unwrap here.